prod-refractr.yml

```diff
@@ -142,6 +142,7 @@ refracts:
 - foundation.mozilla.org/:
   - mozillafoundation.com
   - mozillafoundation.net
+  - surveillance.mozilla.org
```

Comment:
```suggestion
  # SE-1627
  - surveillance.mozilla.org
```

Reply:
adding this comment to go ahead and set up the deploy, since i need that to go out today. sorry, i'd normally wait!